### PR TITLE
Add roocs-grids

### PR DESCRIPTION
**Pull request by sol1105**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by cehbrecht** [comment]
@sol1105 I confirm to be a maintainer of the roocs-grid package.

**Comment by Zeitsperre** [comment]
@sol1105 Sure! I don't mind helping out here!

**Comment by sol1105** [comment]
@Zeitsperre @cehbrecht Great, thank you :+1:

File: recipes/roocs-grids/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python >=3.8
+
+test:
```
Comment (xhochy) [inline recipes/roocs-grids/meta.yaml:26]: Please also add this to ensure no new requirements will be missed in the future.
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check 
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by sol1105** [comment]
@xhochy: Thank you for your feedback. Maybe you can help me as well with the following issue: Windows fails to extract the `tar.gz` - file, which causes the checks to fail:
```
Source cache directory is: C:\bld\src_cache
Downloading source to cache: roocs_grids-0.1.1_30f48caa04.tar.gz
Downloading https://pypi.io/packages/source/r/roocs-grids/roocs_grids-0.1.1.tar.gz
Success
Extracting download
Traceback (most recent call last):
  File "C:\Miniforge\lib\shutil.py", line 816, in move
    os.rename(src, real_dst)
FileExistsError: [WinError 183] Cannot create a file when that file already exists: 'C:\\bld\\tmpl9uye9no\\roocs_grids-0.1.1\\aux' -> 'C:\\Users\\VSSADM~1\\AppData\\Local\\Temp\\tmpyip3kvr0\\aux'
```

I tried to disable Windows build by adding:
```
build:
    ...
    skip: True  # [win]
```
Then the windows check is successful, but the linting check fails.

**Comment by cehbrecht** [comment]
@sol1105 can we do it like in the `cdo` package?

**Comment by sol1105** [comment]
@cehbrecht When requiring a minimum python version (`python>=3.8`),  `noarch: python` is required.
Yesterday I tried the cdo case:
```
build:
  script: {{ PYTHON }} -m pip install . -vv
  number: 0
  skip: True # [win]
```
and whatever fix googling suggested for the minimum python version:
```
build:
  script: {{ PYTHON }} -m pip install . -vv
  number: 0
  skip: True  # [win]
  skip: True  # [py<38]
```
but both failed the checks.

@Zeitsperre Ok, will do :+1:

File: recipes/roocs-grids/meta.yaml [hunk 2]
```diff
@@ -19,6 +19,7 @@ requirements:
   host:
     - python
     - pip
+    - flit
     - pytest-runner
```
Comment (Zeitsperre) [inline recipes/roocs-grids/meta.yaml:23]: I think we can drop `pytest-runner`. I think it's old/obsolete.

**Comment by sol1105** [comment]
@Zeitsperre Yes, that worked out, thanks.

@xhochy It would be ready for another review

File: recipes/roocs-grids/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: 7dbf178314950b0c0160032412677141106701c38141d2481c0e622c7b3efed0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python
```
Comment (xhochy) [inline recipes/roocs-grids/meta.yaml:20]: This should be the same as in run.
```suggestion
    - python >=3.8
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by sol1105** [comment]
@xhochy Okay, thank you, I made the requested change

**Comment by sol1105** [comment]
@conda-forge/help-python, ready for review

Thanks

**Review by xhochy — APPROVED** [review]

**Pull request merged**